spec/features/people_spec.rb

```diff
@@ -33,6 +33,7 @@
       expect(page).to have_content("Andrzej")
       expect(page).to have_content("Kapusta")
       expect(page).to have_content("a.kapusta@gmail.com")
+      expect(page).to have_content('/uploads/person/photo')
```

Comment:
Żeby to zadziałało, to trzeba dostarczyć jakiś plik w formularzu tworzenia osoby. Ponadto raczej trzeba sprawdzić czy jest wyświetlany plik dany w formularzu a nie np. domyślny.